notebooks/speculative-sampling/speculative-sampling.ipynb

```diff
@@ -75,9 +75,17 @@
    "metadata": {
```

Comment:
<p>porbably better to add printing formatting for keeping only several digits after comma</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2609/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>